string library_path
---
string result
